git/git.go

```diff
@@ -615,6 +615,32 @@ func GitAndRootDirs() (string, string, error) {
 	return absGitDir, absRootDir, nil
 }
 
+func PathFromRoot(pattern string) ([]string, error) {
```

Comment:
I considered making the function just return one path or error if there are more, but it seems to me the evolution of the lock commands will/should include supporting multiple files at a time (e.g. `git lfs lock a.jpg b.jpg`, or `git lfs lock *.zip` so returning multiple paths directly and letting be up to the caller to decide about supporting multiple files or just one.

On the other hand, the function as it stands will match many more files than one would initially guess, so if one passes "a.zip" as argument I think the function will return "a.zip" files in folder and subfolders. This will only cause more trouble if git-lfs starts supporting wildcard arguments ("*" would lock the entire repo!). So this needs more design attention, but right now seems like an improvement over the previous functionality and I didn't want to complicate it with more API points.